### ISSUE TITLE
CLI fix multiple termination string 

### ISSUE THREAD

**Issue by ZiGaMi** [GeneralEmbeddedCLibraries/cli#8]
When using 
```C
/**
 * 	Termination string
 *
 * 	@note	Each "cli_printf" will have termination string appended to
 * 			its back.
 *
 * 			If not used leave empty. E.g. ""
 */
#define CLI_CFG_TERMINATION_STRING				((const uint8_t*) ( "\r\n" ))
```

Problem: 
![image](https://user-images.githubusercontent.com/33954294/203108219-15c0fd4f-883a-4aac-9794-b345a2a3f600.png)

CLI parser detects two command instead of single!!!

**Comment by ZiGaMi** [comment]
Problem in code:
![image](https://user-images.githubusercontent.com/33954294/203108433-fe083596-e921-4637-9da9-11c0eded0dfc.png)

**Comment by ZiGaMi** [comment]
Solved in 76c7b02
